.pipelines/pipeline.user.windows.official.redfield.yml

```diff
@@ -0,0 +1,80 @@
+environment:
+  host:
+    os: 'windows'
+    flavor: 'server'
+    version: '2019'
+  runtime:
+    provider: 'appcontainer'
+    image: 'cdpxwin1809.azurecr.io/global/vse2019:latest'
+    source_mode: 'link'
+
+version:
+  name: 'ApplicationInsights-dotnet'
+  major: 0
+  minor: 0
+  system: 'custom'
+
+signing_options:
+  profile: 'external_distribution'
+
+restore:
+  commands:
+    - !!defaultcommand
+      name: 'Hello World'
+      command: '.pipelines\00_helloworld.cmd'
+      arguments: 'official redfield'
+      fail_on_stderr: false
+    - !!defaultcommand
+      name: 'Show .NET versions'
+      command: '.pipelines\00_list_versions.cmd'
+      fail_on_stderr: false
+    - !!defaultcommand
+      name: 'Install dependencies'
+      command: '.pipelines\01_restore.cmd'
+      fail_on_stderr: false
+
+build:
+  commands:
+    - !!buildcommand
+      name: 'Dotnet Build'
+      command: '.pipelines\02_build.cmd'
```

Comment:
nit: many of these commands (e.g. dotnet) are platform independent, consider getting rid of the cmd file.

e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/66d2621170c614313c81fc82e4f6c79218244974/.github/workflows/linux-ci.yml#L44

Reply:
I would have GREATLY preferred this approach.
I went to office hours to ask this exact question. OneBranch does not permit in-line scripts as a rule!

In all of our github builds, i'm doing everything in-line:
https://github.com/microsoft/ApplicationInsights-dotnet/blob/e8eb5204d2757f7443c43cb546d6e3f03f201ae3/.github/workflows/build-and-test-BASE.yml#L47